element-hq/element-x-android
# Avatar cluster for DM

## Type of change


## Content


Let DM detail to render both user and other user Avatar. Also render "Encrypted" badges in DM.

|Before|After|
|-|-|
|<img width="280" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/0391882d-752e-4966-9346-6d7f75966170">|<img width="280" alt="image" src="https://github.com/element-hq/element-x-android/assets/3940906/b7320ee8-9fac-441a-a620-b5d17995f5bb">|

Also fix some rendering avatar issue.

Figma: https://www.figma.com/design/A2pAEvTEpJZBiOPUlcMnKi/Settings-%2B-Room-Details-(new)?node-id=1787-54050&m=dev

## Motivation and context


Parity with EXI

## Screenshots / GIFs


See recorded screenshot

## Tests



- Open a DM
- Navigate to the room settings
- Observe that 2 avatars are rendered

## Tested devices

- OS version(s):

## Checklist
